ui/components/ui/sender-to-recipient/sender-to-recipient.component.js

```diff
@@ -98,6 +101,7 @@ SenderAddress.propTypes = {
 };
 
 export function RecipientWithAddress({
+  recipientAddress,
```

Comment:
Do we need the non-normalized valu? Does the `Name` component and hook not normalize any input?

Reply:
Moved to #22515